Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15.12 as builder
+FROM golang:1.16.3 as builder
```

Comment:
On line 11: do not call `go mod vendor` during build. That gives you a different `vendor/` than in this repo and can lead to surprises and frustration why a wrong lib is used when `vendor` has the right one.

Reply:
On line 12 below: can you just call `make`? There should be only one canonical way how to build the binary.